bootstrap_resume_sass/scss/main.scss

```diff
@@ -0,0 +1,231 @@
+$primary-color: green;
+$font-size: 1.5vw;
+$third: 33.333%;
+$ten-pixel: 10px;
```

Comment:
Since you are using multipliers like 5, 4, 3, 1.5. I would say this is an overkill and not quite helpful. Would either go with few predefined spacing values named small-spacing, medium-spacing etc. or keep them hardcoded in the code if they are specific to a single selector.